src/services/mentees.js

```diff
@@ -925,8 +925,42 @@ module.exports = class MenteesHelper {
 						relatedOrgs = userOrgDetails.data.result.related_orgs
 					}
 					if (visibilityPolicy === common.ASSOCIATED) {
-						organization_ids.push(...relatedOrgs)
+						const associatedAdditionalFilter =
+							filter_type.toLowerCase() == common.MENTEE_ROLE
```

Comment:
@adithyadinesh0412  filter_type.toLowerCase() at every condition is not good, set it at the beginning

Reply:
@adithyadinesh0412 And rename this filter_type variable to filterType